### ISSUE TITLE
Instance norm during evaluation

### ISSUE THREAD

**Issue by HectorAnadon** [junyanz/pytorch-CycleGAN-and-pix2pix#395]
Hello,

I notice that during training, when sampling the images the model is not in eval mode https://pytorch.org/docs/stable/nn.html#torch.nn.Module.eval. Then, as track_running_stats is True in InstanceNorm, the model should behave differently when it is training or evaluating. I might have missed something.

**Comment by naoto0804** [comment]
I'm also curious about this. When I visit this repository a year ago, there was no `track_running_stats` in PyTorch. Does `track_running_stats` affect the performance?

**Comment by taesung89** [comment]
I agree. Let's set it to false. @HectorAnadon thanks for pointing this out! We really appreciate it.

**Comment by junyanz** [comment]
I updated the flag.
